opentasks/src/main/java/org/dmfs/tasks/utils/RecentlyUsedLists.java

```
@@ -105,9 +118,16 @@ public static Long getRecentFromList(Context context, List<Long> allowedLists)
      */
     public static void use(Context context, Long listId)
     {
-        List<Long> lists = getList(context);
-        lists.remove(listId); // does nothing, if "listId" is not in "lists"
-        lists.add(0, listId);
-        setList(context, lists);
+        // 'null' should not come here but happened before: https://github.com/dmfs/opentasks/issues/562
+        if (listId == null)
```

This is the opposite of "fail fast" ;-)